src/components/NewBeer.js

```diff
@@ -0,0 +1,121 @@
+import React, {useState} from 'react'
+import axios from 'axios'
+import Header from './Header'
+
+const initialState = {
+    name : "",
+    tagline : "",
+    description : "",
+    first_brewed : "",
+    brewers_tips : "",
+    attenuation_level: 0,
+    contributed_by :""
+  };
+
+  const NewBeer = () =>{
+    const [formState, setFormState] = useState(initialState);
+
+    
+    const handleInputChange = (event) => {
+        let inputValue = event.target.value;
+        let inputName = event.target.name;
+        let inputType = event.target.type;
+    
+        if(inputType === "number") {
+          inputValue = parseInt(inputValue)
+        };
+    
+        setFormState({ ...formState, [inputName]: inputValue});
+    
+      };
+      
+function addNewBeer(event) {
+  event.preventDefault();
+  axios
+      .post('https://ih-beers-api2.herokuapp.com/beers/new', formState)
+      .then(() => {
+        console.log(`New beer added`)
+
```

Comment:
You can improve the user experience if you set the form back to it's initial state after adding the beer.
```
setFormState(initialState)